app/assets/stylesheets/components/_flexbox.scss

```diff
@@ -1,6 +1,6 @@
 .flex-container {
```

Comment:
avoid using "flex" in CSS class declaration as potentially conflicting

prefer smthg like card "card-stripe-body", or alternatively create a <div> in the HTML code and add the bootstrap classes directly "d-flex justify-content-between ..etc"